sdk/videoanalyzer/video-analyzer-edge/review/video-analyzer-edge.api.md

```diff
@@ -0,0 +1,745 @@
+## API Report File for "@azure/video-analyzer-edge"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public
+export interface CertificateSource {
+    "@type": "#Microsoft.VideoAnalyzer.PemCertificateList";
+}
+
+// @public (undocumented)
+export type CertificateSourceUnion = CertificateSource | PemCertificateList;
+
+// @public
+export type CognitiveServicesVisionProcessor = ProcessorNodeBase & {
+    "@type": "#Microsoft.VideoAnalyzer.CognitiveServicesVisionProcessor";
+    endpoint: EndpointBaseUnion;
+    image?: ImageProperties;
+    samplingOptions?: SamplingOptions;
+    operation: SpatialAnalysisOperationBaseUnion;
```

Comment:
`SpatialAnalysisOperationBaseUnion` seems like a pretty bad name, can we call it something better? Maybe just `SpatialAnalysisOperation` ?

Reply:
Hm this is coming from our swagger. Not sure we can change it for this realse but for the next swagger release we can rename

Reply:
Sure, you can also probably use a swagger transform if you don't want to update it in the swagger directly